dnd5e.mjs

```diff
@@ -190,8 +192,9 @@ Hooks.once("ready", function() {
   _configureValidationStrictness();
 
   // Apply custom compendium styles to the SRD rules compendium.
-  const rules = game.packs.get("dnd5e.rules");
-  rules.apps = [new applications.journal.SRDCompendium(rules)];
+  const rules = {collection: game.packs.get("dnd5e.rules")};
+  const isV10 = !foundry.utils.isNewerVersion(game.version, "10.291")
```

Comment:
Previously we've attached this to the dnd5e object to make it easier to reference elsewhere. We also should probably use `game.release.generation` rather than checking an explicit version like `291` to make sure this check stays accurate even in the face of a closing v10 release.
```
game.dnd5e.isV10 = game.release.generation < 11;
```

Reply:
I appended it to the `game.dnd5e` object, an also version gated the `_getSourceName` call.